pyaedt/application/Analysis3D.py

```diff
@@ -1034,3 +1034,87 @@ def flatten_3d_components(self, component_name=None, purge_history=True, passwor
         if not self.design_type == "Icepak":
             self.mesh._refresh_mesh_operations()
         return True
+
+    def identify_touching_conductors(self, object_name=None):
+        # type: (str) -> dict
```

Comment:
@maxcapodi78 Do we start to add type hints in the methods signature for PyAEDT?

Reply:
@MaxJPRey I noticed that improves the quality of output documentation and helps in autocomple in IDE